index.html

```diff
@@ -1,48 +1,285 @@
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <ul>
+        <li>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-solid fa-m"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-solid fa-m"></i></a>
+        </li>
+      </ul>
+    </section>
+
+
+    <section class="work-section">
+      <h1>My Recent Works</h1>
+      <div class="img-container">
+        <img src="./images/image1.png" alt="" />
+        <h2>Multi-Posts Stories</h2>
         <p>
-            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-don’t hestiate to contact me.
-        </p>
-        </div>
-        <ul id="social-media">
-            <li><a href="#"><img src="images/github.svg" id="github"></a></li>
-            <li><a href="#"><img src="images/linkedin.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/Hand.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/M.svg" alt=""></a></li>
-        </ul>
-        
-      </section>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul>
+          <li>
+            <a href="#">css</a>
+          </li>
+          <li>
+            <a href="#">html</a>
+          </li>
+          <li>
+            <a href="#">Bootstraps</a>
+          </li>
+          <li>
+            <a href="#">Ruby</a>
+          </li>
+        </ul>
+        <button>see project</button>
+      </div>
+    </section>
+    <section class="cards">
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>

```

Comment:
- [ ] From `375px` view port and above, your `See project` button should look exactly like what your have on the Figma template. Anything below `375px` view port can be ignored. But in your own case, from `375px` to `398px` view port, this is how the button appears on mobile. :point_down: 

![btn-responsiveness](https://user-images.githubusercontent.com/26597432/200126083-1f6175ea-3eb0-43c1-9250-2be6dfffd9e6.png)

- From `399px` view port above, your `See Project` button appears as expected.

![from-another-vp](https://user-images.githubusercontent.com/26597432/200126272-a5e0c813-6bca-4ff3-80cd-5e070c4d8a20.png)

- [ ] Kindly make it appear as we have it on the Figma template from `375px` view port and above.